test/integration/wrapEth.ts

```diff
@@ -49,7 +47,6 @@ import { getCanonicalWrappedEther } from "../utils/tokens";
       asset: wethAsset,
     });
     const finalWeth = formatEther(finalWethBalance.toString());
-    console.log(`Final Weth: ${finalWeth}`);
     assert(
       Number(startingWeth) === Number(finalWeth),
       "Balances should match."
```

Comment:
If we are removing this console log, can we add the balances to the assert failure message?

Reply:
yes i will swap assert for expect so it will log the difference

Reply:
actually wouldnt assert also do the same?

Reply:
oh needs assert.equal, will update